main.rb

```diff
@@ -1,40 +1,12 @@
 require './app'
+require './display'
+require './display_option'
 
 class Main
   def initialize
     @app = App.new
   end
 
```

Comment:
- [ ] It would be nice if you ask your partner to commit to this pull request as well to make your project follow the requirements 👍
- [ ] It would be great if you make your branch name more descriptive such as `feature/refactor_code` to make your pull request descriptive and professional 😊